### PR TITLE
Altering rrd2whisper.py to use listcomp instead of filter() for py3 c…

### LINKED ISSUE #278: rrd2whisper len(filter(...)) fails on python3
Hello,

len() on a filter() is not supported in python3, causing rrd2whisper to fail, solely because of the print() at rrd2whisper.py#161
https://github.com/graphite-project/whisper/blob/762cab790bac1f4a01ea4ff52f1e027ce2fbf195/bin/rrd2whisper.py#L161

A few options I see:
- Replace the use of filter with a list comprehension that provides the filter.  This comes with the memory tradeoff of abandoning filter()
- Change the print to use len(values) or len(timestamps) (or the lesser of the 2 if they are different for some reason), with the caveat that this overestimate will count None values that are later excluded by the filter()

If anyone can express a preference, I will add this to a PR I am working on for another tweak/fix to rrd2whisper

### PR DESCRIPTION
**Pull request by FliesLikeABrick**
…ompatibility with len(), to fix #278

### CODE REVIEW

File: bin/rrd2whisper.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/home/rrawdon/git/whisper-env/bin/python3
```
Comment (piotr1212) [inline bin/rrd2whisper.py]: This shouldn't be changed

Comment (FliesLikeABrick) [reply bin/rrd2whisper.py]: oh geez, that's what I get for working off of an installed copy rather than my git copy.  Will update shortly.

Comment (piotr1212) [reply bin/rrd2whisper.py]: Check out setuptools develop mode or with `pip install -e .`. You won't need for copies any more.

**Comment by piotr1212** [comment]
I reverted it, thanks for fixing this,

**Pull request merged**